deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/circuitbreaker/CircuitBreakerConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkiverse.openapi.generator.deployment.circuitbreaker;
+
+import java.util.List;
+
+public final class CircuitBreakerConfiguration {
+
+    private final List<CircuitBreakerClassConfiguration> classConfigurations;
+
+    public CircuitBreakerConfiguration(List<CircuitBreakerClassConfiguration> classConfigurations) {
+        this.classConfigurations = List.copyOf(classConfigurations);
+    }
+
+    public List<CircuitBreakerClassConfiguration> getClassConfigurations() {
+        return classConfigurations;
+    }
+
+    CircuitBreakerClassConfiguration getClassConfiguration(String className) {
```

Comment:
I think is better to make this method public

Reply:
The implemenation with map will be classConfiguration.get(className), contant rather than lineal